setup.py

```diff
@@ -69,7 +69,7 @@
 
 setup(
     name="data_pipelines_cli",
-    version="0.25.3",
+    version="0.26.0",
```

Comment:
I think it's redundant here - version will be bumped in the PR into main created by the github actions bot.